include/ui/memory-component-presenter.hpp

```diff
@@ -98,6 +99,15 @@ class MemoryComponentPresenter : public QAbstractListModel {
    */
   QHash<int, QByteArray> roleNames() const Q_DECL_OVERRIDE;
 
+  /**
+   * returns the number of bytes to be displayed in one memory cell.
```

Comment:
The `\brief` part (`\brief`) is implicit here should fit on a single line. You can put more detailed description in the )(implicit) `\detail` part (after a newline). Also, start the sentence with a capital character